zone/zone.cpp

```diff
@@ -1005,6 +1008,9 @@ Zone::Zone(uint32 in_zoneid, uint32 in_instanceid, const char* in_short_name)
 		m_graveyard_id = z->graveyard_id;
 		m_max_clients  = z->maxclients;
 
+		SetIdleWhenEmpty(z->idle_when_empty);
+		SetSecondsBeforeIdle(z->seconds_before_idle);
```

Comment:
This is fine to leave here but since you are the zone class you could have used them since they are private and belong to zone